src/kuzya.h

```diff
@@ -142,6 +142,7 @@ private slots:
         void slotModificationChanged(bool);
 
         void slotSetFileSuffix(QStringList);
+        void slotSetFileSuffix(QString suffix);
```

Comment:
Why we have 2 methods here with different parameters? Do we really need all of them?